### ISSUE TITLE
Importing an enum from a default export does not work

### ISSUE THREAD

**Issue by philiiiiiipp** [dotansimha/graphql-code-generator#3180]
**Describe the bug**
Importing a enum from a default export results in 
```
import { UserStatus } from './src/Status';
```

instead of 
```
import { default as UserStatus } from './src/Status';
```

**To Reproduce**

1. My GraphQL schema:

```graphql
enum UserStatus {
  New
  Old
}
```

`Status.ts`
```ts
export enum UserStatus {
  New,
  Old
}

export default UserStatus
```

3. My `codegen.yml` config file:

```yml

schema:
  - ./src/schema.graphql
generates:
  ./src/types.d.ts:
    plugins:
      - typescript
      - typescript-operations
    config:
      enumValues:
        UserStatus: ./src/Status#default
```

**Environment:**

- OS: 
- @graphql-codegen/cli@1.9.1 
- NodeJS: v12.13.1

Using the `export` directly works fine, might be related to #1997

**Comment by dotansimha** [comment]
Hi @philiiiiiipp 
I fixed this issue in this PR: https://github.com/dotansimha/graphql-code-generator/pull/3189

Can you please try the alpha version? `1.9.2-alpha-61f0b7ca.85`

**Comment by dotansimha** [comment]
Available in `v1.10.0`.
